jobs/JGFS_ATMOS_CYCLONE_TRACKER

```diff
@@ -138,6 +135,9 @@ elif [ $machine = LUNA -o $machine = SURGE -o $machine = WCOSS_C ] ; then # For
 elif [ $machine = VENUS -o $machine = MARS -o $machine = WCOSS_DELL_P3 ] ; then # For DELL
   machine=dell
   mpirun -n 1 ${USHens_tracker}/extrkr_gfs.sh ${loopnum} ${cmodel} ${CDATE} ${pert} ${DATA}
+elif [ $machine = WCOSS2 ] ; then # For WCOSS2
+  machine=wcoss2
+  mpirun -n 1 ${USHens_tracker}/extrkr_gfs.sh ${loopnum} ${cmodel} ${CDATE} ${pert} ${DATA}
```

Comment:
I think the correct usage on wcoss2 is `mpiexec` and not `mpirun`.  Please confirm.

Reply:
Really these should all be using the launcher variable set by the ${machine}.env file and not have all this machine-branching code here at all.

Reply:
1. mpirun is ok to use because host is assigned. It is located in:
     /pe/intel/compilers_and_libraries_2020.4.304/linux/mpi/intel64/bin/mpirun
    Program run without issue see job log:
      /lfs/h2/emc/global/noscrub/Lin.Gan/git/feature-ops-wcoss2/jobs/gfs_emc_vrfy_00.o5065799
2. Using launcher variable in jjob is not within the scope of this work. This job exist in workflow not a new job in wcoss2. Looking at the env file, I do not see every job within workflow are using it. Recommend to handle this request in a separate GitHub issue.

Reply:
1.  Per Carolyn Pasti's presentation on WCOSS2, the correct usage is `mpiexec` and not `mpirun`.
Please see the example from slide 53 from [this](https://www2.emc.ncep.noaa.gov/forms/training/WCOSS2gettingStarted.pdf) presentation.
2. Use of `launcher` is out of the scope of this work.  An issue can be created to resolve this.

Reply:
I do not see problem of using mpirun. The code manager Jiayi should be contacted, if request for change. He is ok with it and checked the output found no issue.

Reply:
Please work w/ whoever it is to bring up to the correct and acceptable usage.  `mpirun` is unacceptable per WCOSS2 instructions.

Reply:
Please use mpiexec, all wcoss2 codes are using that. We have had extensive discussions with NCO on this already

Reply:
Removed the use of mpirun.